packages/database/src/core/SyncPoint.ts

```diff
@@ -94,23 +94,12 @@ export class SyncPoint {
     }
   }
 
-  /**
-   * Add an event callback for the specified query.
-   *
-   * @param {!Query} query
-   * @param {!EventRegistration} eventRegistration
-   * @param {!WriteTreeRef} writesCache
-   * @param {?Node} serverCache Complete server cache, if we have it.
-   * @param {boolean} serverCacheComplete
-   * @return {!Array.<!Event>} Events to raise.
-   */
-  addEventRegistration(
+  getView(
     query: Query,
-    eventRegistration: EventRegistration,
     writesCache: WriteTreeRef,
     serverCache: Node | null,
     serverCacheComplete: boolean
-  ): Event[] {
+  ): View {
     const queryId = query.queryIdentifier();
     let view = this.views.get(queryId);
```

Comment:
As some lint tooling says: 'view' is never reassigned. Use 'const' instead.

Or assign it in the if block.